samples/Calling/eslint.config.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+import { globalIgnores } from 'eslint/config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
+      'headers/header-format': 'off',
+      'jsdoc/require-jsdoc': 'off'
+    }
+  },
+  {
+    linterOptions: {
+      reportUnusedDisableDirectives: false
```

Comment:
Should this be here?

Reply:
yes there are both inline rules and global rules so we want to disable them here unless we want to either remove all of the inline rules or remove the global rule and add in more inline rules.